src/utils/string/StringUtils.js

```diff
@@ -163,6 +162,7 @@ const displayFormattedSearchLocationsV2 = (searchQuery, locationObject) => {
   return null;
 };
 
+// eslint-disable-next-line consistent-return
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The ESLint rule `consistent-return` is disabled here, which could lead to inconsistent behavior. Instead of disabling this rule, consider returning a consistent type of value from the function.
```diff
- // eslint-disable-next-line consistent-return
+ // No changes needed here
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->